client/reader/reader-start/index.jsx

```diff
@@ -0,0 +1,26 @@
+// External dependencies
```

Comment:
We call this file `main.jsx` by convention (the top level component that renders `<Main>`) and allows to reserve `index.js` as the module entry point for sections.js.